requirements/edx/base.txt

```diff
@@ -79,14 +79,14 @@ python-social-auth==0.2.12
 pytz==2015.2
 pysrt==0.4.7
 PyYAML==3.10
-requests==2.7.0
+requests==2.9.1
 requests-oauthlib==0.4.1
 scipy==0.14.0
 Shapely==1.2.16
 singledispatch==3.4.0.2
 sorl-thumbnail==12.3
 sortedcontainers==0.9.2
-stevedore==0.14.1
+stevedore==1.10.0
```

Comment:
@andy-armstrong you're more familiar with our uses of stevedore can you take a look at the changelog to make sure there is nothing of concern.  I looked over it briefly and it seems to be mostly changes related to adding support newer python and better packaging.